<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tomar un vuelo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(174, 198, 206);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Tomar un Vuelo</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Elige un destino</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QComboBox" name="DestinoV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seleccione opcion:</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cochamo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Puelo_Bajo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Contao</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rio_Negro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pupelde</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chepu</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aycara</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pillán</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Renihue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Isla_Quenac</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Palqui</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaiten</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Santa_Barbara</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Elige un avion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QComboBox" name="TipoAvion">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seleccione opción:</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VCessna310</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VCessna208_Caravan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VLET_410_UVP_E20</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnInfoAviones">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver informacion de aviones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmarV">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
